tests/lib/Search/Query/Utils/Stub/TestCriterion.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Tests\Solr\Search\Query\Utils\Stub;
+
+use Ibexa\Contracts\Core\Repository\Values\Content\Query\Criterion;
+use Ibexa\Contracts\Core\Repository\Values\Content\Query\Criterion\Value;
+
+final class TestCriterion extends Criterion
+{
+    public function __construct(?string $target = null, ?string $operator = null, $value = [], ?Value $valueData = null)
```

Comment:
Test criterion is never initialized with data. You can inline proper values.

```suggestion
    public function __construct()
```

Reply:
Fixed: https://github.com/ibexa/solr/pull/77/files#diff-be34c28f75856edd7547e70b50888be3da018caee383501c4a3b69edcbeb0216